interfaces/udev/backend.go

```diff
@@ -69,6 +72,7 @@ func (b *Backend) Setup(snapInfo *snap.Info, opts interfaces.ConfinementOptions,
 		return fmt.Errorf("cannot obtain udev specification for snap %q: %s", snapName, err)
 	}
 	content := b.deriveContent(spec.(*Specification), snapInfo)
+	subsystemTriggers := spec.(*Specification).GetTriggeredSubsystems()
```

Comment:
NOTE: this is not assigning to the global variable with the same name (because of :=), is this intentional?

Reply:
This is a holdover from some ideas I had to try to get things to work on disconnect. With the current implementation, I think it is correct to use ':=' and I'll take a look at cleaning up the global.